### ISSUE TITLE
Buffer Doesn't Restart After Internet Connection Drop/Reconnect

### ISSUE THREAD

**Issue by bitnom** [jaruba/PowderPlayer#117]
If while streaming a torrent, the Internet connection goes down momentarily, the stream will not continue to buffer once the Internet connection is restored. The torrent has to be reloaded and then seek to where you left off. This can be especially annoying in situations where you forget the time you were at in the clip and when you face pulseaudio issues where pulseaudio and powder must be restarted to get audio working again.

Basically though, it's an error correcting sort of feature you'd expect. Network conditions aren't always great at all times.

**Comment by jaruba** [comment]
You can right click the video and select "Force Download", same goes for both v0.98 and v1.00 (this option is also available in the Torrent Dashboar settings, in case you're not streaming anything but simply downloading a torrent)

"Force Download" starts the search for new peers and reconnects to all previously discovered peers regardless of the reason that disconnected from those peers in the first place. It's a very useful option that helps in many other cases too, such as very old torrents with really low peer count that at one point lost most or all of it's peers or situations in which the torrent is very new and is buffering poorly because it's gaining peers faster then Powder is searching for new ones.

Powder does have an emergency case that checks the peer situation periodically and does "Force Download" on it's own if it considers it must. I am uncertain if this auto-repair logic works correctly in v1.00 but it works in v0.98 for sure, it's still obviously slower then doing "Force Download" manually when you know it's needed.

**Comment by jaruba** [comment]
Did you try to use the feature I suggested? Was it helpful? :)
